### DISCUSSION TITLE
Re Render remove all the model that use teleport from the DOM

### DISCUSSION THREAD

**Discussion by sohailahmad07** [livewire/livewire#6615] [7. Bugs (archived)]
### Livewire version

3.0.1

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. create a simple blade modal component (code is given below) 
2. teleport the content to the body
3. create a simple livewire component
4. Open the modal and rerender the component by using $refresh
5. and voila you reproduce the bug 
6. all the code is give below

### Additional information

Simple Modal Blade Component
```
<div x-data='{show: false}' @keydown.window.esc='show =false'>
    <div @click='show = true'>
        {{ $trigger }}
    </div>

    <template x-teleport='body'>
        <div x-show='show' x-cloak
             class='fixed top-1/2 left-1/2 -translate-y-1/2 -translate-x-1/2 bg-blue-600'>
            {{ $slot }}
        </div>
    </template>
</div>
```

simple livewire component view

```
<div>

    <x-modal>
        <x-slot:trigger>Click ME</x-slot:trigger>
        <div class='py-12 px-6'>
            <button class='text-white' wire:click="$refresh">Dispatch
                Event
            </button>
        </div>
    </x-modal>
</div>
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by robsontenorio** [comment]
@sohailahmad07  did you solved this?
